pkg/iphelpers/iphelpers_test.go

```diff
@@ -571,6 +572,190 @@ var _ = Describe("DecIPAddress operations", func() {
 	})
 })
 
+var _ = Describe("GetIPRange operations", func() {
+	It("creates an IPv4 range properly for 30 bits network address", func() {
+
+		_, ipnet, err := net.ParseCIDR("192.168.21.100/30")
+		ip, _ := AddressRange(ipnet)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
please move this line above, so we expect the error immediately after the call that could have thrown it.

Reply:
That's an interim state from the move code commit IIRC, the final version looks like this, with AddressRange gone:
~~~
  554 var _ = Describe("GetIPRange operations", func() {           
  555     It("creates an IPv4 range properly for 30 bits network address", func() {
  556         _, ipnet, err := net.ParseCIDR("192.168.21.100/30")         
  557         Expect(err).NotTo(HaveOccurred())                                 
  558         firstip, lastip, err := GetIPRange(*ipnet, nil, nil)
  559         Expect(err).NotTo(HaveOccurred())                                       
  560         Expect(fmt.Sprint(firstip)).To(Equal("192.168.21.101"))                                                   561         Expect(fmt.Sprint(lastip)).To(Equal("192.168.21.102"))
  562     }) 
~~~

Reply:
Hm, that still could be improved. Each commit should individually make sense.

But I guess it's OK to let this slide.